package.json

```diff
@@ -1,62 +1,96 @@
-    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+    "lint": "eslint \"{src,apps,dist,test}/**/*.ts\" --fix",
     "test": "jest",
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "test:e2e:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --config test/jest-e2e.json --runInBand",
+    "test:e2e:dev": "yarn test:e2e --watchAll",
+    "version": "conventional-changelog -p angular -i CHANGELOG.md -s && git add CHANGELOG.md"
   },
   "dependencies": {
     "@golevelup/nestjs-modules": "^0.4.1",
-    "@nestjs/common": "^7.0.0",
-    "@nestjs/core": "^7.0.0",
-    "@nestjs/platform-express": "^7.0.0",
-    "md5": "^2.2.1",
-    "path-to-regexp": "^6.1.0",
-    "reflect-metadata": "^0.1.13",
-    "rimraf": "^3.0.2",
-    "rxjs": "^6.5.4"
+    "md5": "^2.2.1"
   },
   "devDependencies": {
+    "@commitlint/cli": "^8.3.5",
+    "@commitlint/config-conventional": "^8.3.4",
     "@nestjs/cli": "^7.0.0",
+    "@nestjs/common": "^7.0.0",
+    "@nestjs/core": "^7.0.0",
+    "@nestjs/graphql": "^7.4.1",
+    "@nestjs/platform-express": "^7.0.0",
+    "@nestjs/platform-fastify": "^7.1.1",
+    "@nestjs/platform-socket.io": "^7.1.2",
+    "@nestjs/platform-ws": "^7.1.2",
     "@nestjs/schematics": "^7.0.0",
     "@nestjs/testing": "^7.0.0",
+    "@nestjs/websockets": "^7.1.2",
     "@types/express": "^4.17.3",
-    "@types/jest": "25.1.4",
+    "@types/jest": "25.2.3",
     "@types/md5": "^2.2.0",
-    "@types/node": "^13.9.1",
+    "@types/node": "^14.0.11",
+    "@types/socket.io": "^2.1.8",
+    "@types/socket.io-client": "^1.4.33",
     "@types/supertest": "^2.0.8",
-    "@typescript-eslint/eslint-plugin": "^2.23.0",
-    "@typescript-eslint/parser": "^2.23.0",
-    "eslint": "^6.8.0",
+    "@typescript-eslint/eslint-plugin": "^3.1.0",
+    "@typescript-eslint/parser": "^3.1.0",
+    "conventional-changelog-cli": "^2.0.31",
+    "cz-conventional-changelog": "^3.1.0",
+    "eslint": "^7.2.0",
     "eslint-config-prettier": "^6.10.0",
     "eslint-plugin-import": "^2.20.1",
-    "jest": "^25.1.0",
+    "graphql": "^15.0.0",
+    "graphql-tools": "^6.0.5",
+    "husky": "^4.2.5",
+    "jest": "^26.0.1",
+    "lint-staged": "^10.2.9",
     "nodemon": "^2.0.4",
-    "prettier": "^1.19.1",
+    "prettier": "^2.0.5",
+    "reflect-metadata": "^0.1.13",
+    "rimraf": "^3.0.2",
+    "rxjs": "^6.5.4",
+    "socket.io": "^2.3.0",
     "supertest": "^4.0.2",
-    "ts-jest": "25.2.1",
-    "ts-loader": "^6.2.1",
+    "ts-jest": "26.1.0",
+    "ts-loader": "^7.0.5",
     "ts-node": "^8.6.2",
     "tsconfig-paths": "^3.9.0",
     "typescript": "^3.7.4"
   },
+  "peerDependencies": {
+    "@nestjs/common": "^7.0.0"
+  },
```

Comment:
Regarding the peer deps: this is what I have locally on develop atm:

```
  "peerDependencies": {
    "@nestjs/common": "^7.0.0",
    "@nestjs/core": "^7.0.0",
    "reflect-metadata": "^0.1.13"
  },
```
shouldn't it be these?

Reply:
I believe `@nestjs/common` already has `reflect-metadata` and `@nestjs/core` as peerDeps, along with `rxjs` at the very least, but I'll double check this.

Reply:
Common has `relfect-metadata` and `rxjs` as peer deps. `core` has `common`, `relfect-metadata` and `rxjs` as peer deps. Should we just add `@nestjs/common`, `@nestjs/core`, and `reflect-metadata` as peer deps? We aren't using `rxjs` explicitly, so I don't think it should be on this package to have a dependency on it